pkg/db/factory.go

```diff
@@ -17,7 +18,7 @@ func NewConnectionFactory(configFile string, migrate bool, debug bool) (Connecti
 
 	encKey, err := readEncryptionKey()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "error reading encryption key")
```

Comment:
Would recommend to enhance directly the error in `readEncryptionKey` instead wrapping it here

Reply:
Was just there for debugging. Wrapping is removed since error of called function has already a good user feedback